pkg/envoyfilter/controller.go

```diff
@@ -131,9 +131,14 @@ func (c *Controller) pushEnvoyFilters2APIServer() error {
 		return fmt.Errorf("failed to generate EnvoyFilter: %v", err)
 	}
 
-	existingEnvoyFilters, _ := c.istioClientset.NetworkingV1alpha3().EnvoyFilters("").List(context.TODO(), v1.ListOptions{
-		LabelSelector: "manager=" + constants.AerakiFieldManager,
-	})
+	listNamespace := ""
+	if c.namespaceScoped {
```

Comment:
Envoyfilters can exist in multiple namespaces, not only in the root namespace.

Reply:
update it.